elements/pfelement/src/pfelement.js

```diff
@@ -2,6 +2,10 @@ import { autoReveal } from "./reveal.js";
 const prefix = "pfe-";
 
 class PFElement extends HTMLElement {
+  static get version() {
+    return "$version";
```

Comment:
Is this related to the minification?

Reply:
Whoops, no, I was just playing around with the gulp-plugin-re.  I'll remove it from this PR.  I would like to add the version number somewhere, either as an element property or maybe in an `@preserve` comment at the top of each file ([like](https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js) [many](https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js) [libs](https://cdnjs.cloudflare.com/ajax/libs/vue/2.6.7/vue.min.js) [do](https://cdnjs.cloudflare.com/ajax/libs/react/16.8.3/umd/react.production.min.js)), so if we're ever looking at a source file we can determine what version it is.